src/reqs.c

```diff
@@ -373,7 +373,7 @@ static struct request_s *process_request (struct conn_s *connptr,
                         safefree (url);
                         url = reverse_url;
                 } else if (config.reverseonly) {
-                        log_message (LOG_ERR, 
+                        log_message (LOG_ERR,
```

Comment:
Great!

Now only indent the two following lines one more space and we're done. ;-)